### DISCUSSION TITLE
Galaxy Brain

### DISCUSSION THREAD

**Discussion by ashishkumar90052** [drknzz/GitHub-Achievements#35] [Q&A] [answered]
how can i get Galaxy Brain badge.

**Comment by drknzz** [comment]
Hey, just answer these 2 discussions:
[Discussion 1](https://github.com/drknzz/GitHub-Achievements/discussions/37)
[Discussion 2](https://github.com/drknzz/GitHub-Achievements/discussions/36)

**Comment by v-nhandt21** [comment]
Haha, come here to get Galaxy Brain, nice trick

**Comment by olamiji-8** [comment, marked as answer]
> how can i get Galaxy Brain badge.
Answered a discussion
(got an accepted answer)

**Comment by Kavitate** [comment]
This still work?
